src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person: { name, age, sex, isMarried, partnerName },
+}) => {
+  const partnerStatus = age && <p className="Person__age">I am {age}</p>;
```

Comment:
It's unnecessary to create a variable for this condition, you can do it directly in jsx